lib/router/config-router.js

```diff
@@ -110,7 +110,10 @@ function ConfigRouter(conf, schedulerRegistry) {
     this.disabledProtocols.push('websocket');
   }
 
-  this.reconnect = !conf.getValues('disable_reconnect').enabled;
+  this.reconnect = false;
```

Comment:
I know it's not part of this PR but maybe the configuration should be called "reconnect" which can be optionally configured.
```
reconnect {
  enabled: false,
  interval: 3s,
  or_whatever: "pickles"
}
```